### PR TITLE
Rearm - Add distance setting

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Add a setting to change the distance a vehicle can be rearmed at.

Someone asked for it on slack and I figured that this could indeed be a setting, the default value is set to the current value (20m) minimum at 10 and maximum at 50.

### CODE REVIEW

File: addons/rearm/stringtable.xml [hunk 1]
```diff
@@ -612,5 +612,13 @@
             <Italian>Il rifornimento delle munizioni (-1 per disabilitarlo)</Italian>
             <Russian>Объем боеприпасов для перевооружения (-1 для отмены)</Russian>
         </Key>
+        <Key ID="STR_ACE_Rearm_RearmSettings_distance_DisplayName">
+            <English>Rearm distance</English>
+            <French>Distance de réarmement</French>
```
Comment (dedmen) [inline addons/rearm/stringtable.xml]:
```suggestion
            <French>Distance de réarmement</French>
            <German>Aufmunitionierungs Reichweite</German>
```
Might be too long?
Might also do `Aufmunitionierungsreichweite` but pls no.

File: addons/rearm/stringtable.xml [hunk 2]
```diff
@@ -612,5 +612,13 @@
...
+        </Key>
+        <Key ID="STR_ACE_Rearm_RearmSettings_distance_Description">
+            <English>The maximum distance a vehicle can be rearmed at</English>
+            <French>Distance maximale à laquelle un véhicle peut être réarmé</French>
```
Comment (dedmen) [inline addons/rearm/stringtable.xml]:
```suggestion
            <French>Distance maximale à laquelle un véhicle peut être réarmé</French>
            <German>Die maximale Distanz, über die ein Fahrzeug Aufmunitioniert werden kann</German>
```

File: addons/rearm/stringtable.xml (hunk 1 above)
Comment (alganthe) [reply addons/rearm/stringtable.xml]: definitely too long

**Comment by PabstMirror** [comment]
For `GVAR(level) = 0`: we need to modify distance check in ace_rearm_fnc_rearmEntireVehicle or progress bar will fail immediately
The other distance checks should be fine as level 1/2 you carry the magazine to the vehicle.

File: addons/rearm/stringtable.xml (hunk 1 above)
Comment (dedmen) [reply addons/rearm/stringtable.xml]: Maybe @commy2 has an idea

Comment (shukari) [reply addons/rearm/stringtable.xml]: Aufmunitionierungsreichweite (28)
Nachschubsreichweite (20)
Nachschubsbereich (17)
Aufrüstungsbereich (17)

some ideas...

Comment (alganthe) [reply addons/rearm/stringtable.xml]: first one could clip, the other three would work, just give me the best one and i'll add it.

Comment (dedmen) [reply addons/rearm/stringtable.xml]: Nachschubbereich would be without s, as schubs is pushing someone, would mean after push area.
Personally I'd just keep Rearm probably :D 

```suggestion
            <French>Distance de réarmement</French>
            <German>Aufrüstbereich</German>
```

**Pull request merged**